FhotoFithFriends/src/main/resources/static/js/index.js

```diff
@@ -1,3 +1,19 @@
-// Elements
+// Elements for taking the snapshot
+var canvas = document.getElementById('canvas');
+var context = canvas.getContext('2d');
+var video = document.getElementById('video');
 
-// This is where your Camera and other API logic can go
\ No newline at end of file
+// Get access to the camera!
+if(navigator.mediaDevices && navigator.mediaDevices.getUserMedia) {
+    // Not adding `{ audio: true }` since we only want video now
+    navigator.mediaDevices.getUserMedia({ video: true }).then(function(stream) {
+        //video.src = window.URL.createObjectURL(stream);
+        video.srcObject = stream;
+        video.play();
```

Comment:
Getting camera permission and displaying the feed. There is a bug where it requires you to refresh the page after loading the first time to get the Camera permission prompt. I'm sure someone can figure it out